src/test/descriptor_tests.cpp

```diff
@@ -99,8 +99,17 @@ void Check(const std::string& prv, const std::string& pub, int flags, const std:
                     spend.vout.resize(1);
                     BOOST_CHECK_MESSAGE(SignSignature(Merge(keys_priv, script_provider), spks[n], spend, 0, 1, SIGHASH_ALL), prv);
                 }
-            }
 
+                /* Infer a descriptor from the generated script, and verify its solvability and that it roundtrips. */
+                auto inferred = InferDescriptor(spks[n], script_provider);
+                BOOST_CHECK_EQUAL(!(flags & UNSOLVABLE), inferred->IsSolvable());
```

Comment:
This is a very picky comment, but maybe reverse the arguments so the test more consistently uses the CHECK_EQUAL(actual, expected) form.

Reply:
Done.